File: app/src/main/java/com/example/votesapp/activities/Lista_salas/Sala.java
package com.example.votesapp.activities.Lista_salas;

public class Sala {
    //Atributos
    private String id;
    private String nombreSala;

    public Sala(){

    }

    public Sala (String id, String nombreSala){
        this.setId(id);
        this.nombreSala=nombreSala;


    }
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }




    public String getNombreSala() {
        return nombreSala;
    }

    public void setNombreSala(String nombreSala) {
        this.nombreSala = nombreSala;
    }


}

File: app/src/main/java/com/example/votesapp/activities/lista_salas/Sala.kt
package com.example.votesapp.activities.lista_salas

class Sala {
    //Atributos
    var id: String? = null
    var nombreSala: String? = null

    constructor() {}
    constructor(id: String?, nombreSala: String?) {
        this.id = id
        this.nombreSala = nombreSala
    }
}